src/Catel.Extensions.DynamicObjects/Catel.Extensions.DynamicObjects.Shared/Data/DynamicObservableObjectMetaObject.cs

```diff
@@ -28,6 +31,10 @@ static DynamicObservableObjectMetaObject()
 
             _getValueMethodInfo = typeof(DynamicObservableObject).GetMethodEx("GetValue", bindingFlags).MakeGenericMethod(new[] { typeof(object) });
             _setValueMethodInfo = typeof(DynamicObservableObject).GetMethodEx("SetValue", bindingFlags);
+
+            bindingFlags = BindingFlags.Instance | BindingFlags.NonPublic;
```

Comment:
Not sure if private reflection is allowed on limited platforms. Can't we expose this in a different way?